test/gtest/ucp/test_ucp_wakeup.cc

```diff
@@ -105,3 +211,59 @@ UCS_TEST_P(test_ucp_wakeup, signal)
 }
 
 UCP_INSTANTIATE_TEST_CASE(test_ucp_wakeup)
+
+class test_ucp_tx_wakeup : public test_ucp_wakeup
+{
+public:
+    static ucp_worker_params_t get_worker_params() {
+        ucp_worker_params_t params = test_ucp_wakeup::get_worker_params();
+        params.field_mask |= UCP_WORKER_PARAM_FIELD_EVENTS;
+        params.events = UCP_WAKEUP_TX_COMPLETION;
```

Comment:
Not sure, because get_worker_params is a static method (instantiation requirement) but GetParam() should be non-static (gtest API)
Pls correct if I don't understand your idea